src/sknnr/_base.py

```diff
@@ -104,33 +120,9 @@ def _set_fitted_transformer(self, X, y):
         """Fit and store the transformer."""
         self.transformer_ = self._get_transformer().fit(X, y)
 
-    @property
-    def feature_names_in_(self):
-        return self.transformer_.feature_names_in_
-
-    @property
-    def n_features_in_(self):
-        return self.transformer_.n_features_in_
-
-    def _check_feature_names(self, X, *, reset):
-        """Override BaseEstimator._check_feature_names to prevent errors.
-
-        This check would fail during fitting because `feature_names_in_` stores original
-        names while X contains transformed names. We instead rely on the transformer to
-        check feature names and warn or raise for mismatches.
-        """
-        return
-
-    def _check_n_features(self, X, *, reset):
-        """Override BaseEstimator._check_n_features to prevent errors.
-
-        See _check_feature_names.
-        """
-        return
-
```

Comment:
Thanks for the very detailed commit message on this change.  I like your suggested methods of retrieving the raw (underlying) feature counts and names.  I agree that it wasn't worth continued workarounds that would lead us to *more* complexity.

Reply:
Sounds good, thanks for giving this some thought!